idaes/models/unit_models/heat_exchanger_1D.py

```diff
@@ -489,132 +515,143 @@ def _make_performance(self):
         Returns:
             None
         """
-        shell_units = (
-            self.config.shell_side.property_package.get_metadata().get_derived_units
+        hot_side_units = (
+            self.config.hot_side.property_package.get_metadata().get_derived_units
         )
-        tube_units = (
-            self.config.tube_side.property_package.get_metadata().get_derived_units
+        cold_side_units = (
+            self.config.cold_side.property_package.get_metadata().get_derived_units
         )
 
         # Unit model variables
         # HX dimensions
-        self.d_shell = Var(
-            initialize=1, doc="Diameter of shell", units=shell_units("length")
+        self.d_hot_side = Var(
+            initialize=1, doc="Diameter of hot side", units=hot_side_units("length")
         )
-        self.d_tube_outer = Var(
-            initialize=0.011, doc="Outer diameter of tube", units=shell_units("length")
+        self.d_cold_side_outer = Var(
+            initialize=0.011,
+            doc="Outer diameter of cold side",
+            units=hot_side_units("length"),
         )
-        self.d_tube_inner = Var(
-            initialize=0.010, doc="Inner diameter of tube", units=shell_units("length")
+        self.d_cold_side_inner = Var(
+            initialize=0.010,
+            doc="Inner diameter of cold side",
+            units=hot_side_units("length"),
```

Comment:
@dallan-keylogic I think we need to consider whether all of this actually belongs in the HX1D model - personally I think we probably should have a completely generic HX1D and a derived ShellAndTube1D model (although that would be more disruptive).

However, this gets a bit more complicated in that the new naming system requires knowledge of which side is hot and cold, but shell and tube do not directly map to these (i.e. the shell side can be hot or cold depending on use case). Again, I think that that detail probably should belong in a derived model rather than the base model.